fixtures/soft_assert.py

```diff
@@ -89,8 +100,8 @@ def _soft_assert_cm():
     """
     _thread_locals.caught_asserts = []
     yield _thread_locals.caught_asserts
-    if _thread_locals.caught_asserts:
-        raise SoftAssertionError(_thread_locals.caught_asserts)
+    # if _thread_locals.caught_asserts:
+    #     raise SoftAssertionError(_thread_locals.caught_asserts)
```

Comment:
I'm not sure we want to comment this out do we? Will the raise still happen inside a context manager if we do?